### DISCUSSION TITLE
Is it possible to Emit custom event from client using socketio

### DISCUSSION THREAD

**Discussion by safaldas** [feathersjs/feathers#2558] [Q&A]
All the information in the docs to send events from server to clinet are working correctly and fine. 
I need to send an event from one client to another without writing to the database. 

app.io.emit('poseDetection', keypoints)

**Comment by daffl** [comment]
That is still what [a custom service](https://docs.feathersjs.com/api/services.html) can be used for. As mentioned in the docs, a service does not have to write to the database and can be used for anything, including distributing data to other users.
